server/src/routes/callAutomationEvent.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as express from 'express';
+import { CALLCONNECTION_ID_TO_CORRELATION_ID } from '../utils/callAutomationUtils';
+
+const router = express.Router();
+
+router.post('/', async function (req, res) {
+  try {
+    if (req.body[0].type === 'Microsoft.Communication.CallConnected') {
+      console.log('/automationEvent received', req.body);
+      /**
+       * if the call already exists in the mapping we don't want to start a
+       */
+      if (CALLCONNECTION_ID_TO_CORRELATION_ID[req.body[0].data.callConnectionId]) {
+        console.log('CallConnectionId already exists in mapping');
+
+        res.status(200).end();
+        return;
+      }
+      /**
+       * We want to make a mapping here between the callConnectionId and the correlationId
```

Comment:
nit: comments should use imperative tense
```suggestion
        * Make a mapping here between the callConnectionId and the correlationId
```